lua/lualine/extensions/chadtree.lua

```diff
@@ -0,0 +1,17 @@
+os = require('os')
+
+local function short_cwd ()
+  local cwd = os.getenv('PWD')
+  local home = os.getenv('HOME')
+  return cwd:gsub(home, '~')
```

Comment:
I think we can get away without calling `os` since vim already caches the working directory.

Try this in command mode
```
:substitute(getcwd(),$HOME,'~','')
```